app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterMovementReferenceNumberController.scala

```diff
@@ -114,7 +113,7 @@ class EnterMovementReferenceNumberController @Inject() (
                       )
                     ),
                   isAmend,
-                  getRoutes(numOfClaims, Some(MovementReferenceNumber(Right(MRN(""))))).subKey
+                  getRoutes(journey, Some(MovementReferenceNumber(Right(MRN("")))))
```

Comment:
I'm a bit worried about there being two different ways to calculate the routes. Perhaps a call to discuss when you're free?